eras/conway/test-suite/cardano-ledger-conway-test.cabal

```diff
@@ -54,8 +54,7 @@ library
         cardano-strict-containers,
         microlens,
         tasty,
-        tasty-quickcheck,
-        QuickCheck
+        tasty-quickcheck
```

Comment:
```suggestion
        tasty-quickcheck,
```
Maybe leave the trailing comma so diffs look nicer.

Reply:
Oh, never mind, I forgot we auto-format this now

Reply:
Yep auto-format :slightly_smiling_face: